fastembed/text/e5_onnx_embedding.py

```diff
@@ -0,0 +1,38 @@
+from typing import Type, List, Dict, Any
```

Comment:
Not a fan of the approach of having completely different classes for each model implementation e.g. e5 and JINA here. This makes it harder for beginners to add more models, which is right now simplified to adding it to the models.json and nothing else!

Reply:
if the new model doesn't require a special type of inference, you can add it to the list as well